### PR TITLE
Roles capabilities create/list

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

File: ui/admin/tests/acceptance/roles/create-test.js [hunk 1]
```diff
@@ -11,64 +11,95 @@ import {
...
+        scope: instances.orgScope,
       },
       'withPrincipals'
     );
-    urls.roles = `/scopes/${instances.scopes.org.id}/roles`;
+    urls.roles = `/scopes/${instances.orgScope.id}/roles`;
     urls.role = `${urls.roles}/${instances.role.id}`;
-    urls.newRole = `${urls.roles}/new`;
+    urls.newRoleURL = `${urls.roles}/new`;
+    orgURL = `/scopes/${instances.orgScope.id}`;
```
Comment (gsusmi) [inline ui/admin/tests/acceptance/roles/create-test.js:68]: Any reason `orgURL` cannot be part of `urls` object as `urls.org`? `urls.orgScope` is also defined so wondering if that can be used instead?

File: ui/admin/tests/acceptance/roles/create-test.js [hunk 2]
```diff
@@ -11,64 +11,95 @@ import {
...
-        scope: instances.scopes.org,
+        scope: instances.orgScope,
       },
       'withPrincipals'
     );
-    urls.roles = `/scopes/${instances.scopes.org.id}/roles`;
+    urls.roles = `/scopes/${instances.orgScope.id}/roles`;
     urls.role = `${urls.roles}/${instances.role.id}`;
-    urls.newRole = `${urls.roles}/new`;
+    urls.newRoleURL = `${urls.roles}/new`;
```
Comment (gsusmi) [inline ui/admin/tests/acceptance/roles/create-test.js:67]: Seeing as `newRole` is defined within `urls` object, `newRole` shouldn't need `URL` append to it's name.

File: ui/admin/tests/acceptance/roles/create-test.js (hunk 1 above)
Comment (calcaide) [reply ui/admin/tests/acceptance/roles/create-test.js:68]: +1

File: ui/admin/tests/acceptance/roles/list-test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    instances.role = this.server.create('role', {
+      scope: instances.orgScope,
+    });
+    orgURL = `/scopes/${instances.orgScope.id}`;
+
+    urls.roles = `/scopes/${instances.orgScope.id}/roles`;
+    authenticateSession({});
+  });
+
+  test('can navigate to roles with proper authorization', async function (assert) {
```
Comment (calcaide) [inline ui/admin/tests/acceptance/roles/list-test.js:50]: Minor typo, for consistency with the other test, can we add the word "User"?

**Review by calcaide — APPROVED** [review]
LGTM!

File: ui/admin/tests/acceptance/roles/create-test.js (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/tests/acceptance/roles/create-test.js]: updated to  `urls.orgScope`

**Pull request merged**